react/features/participants-pane/components/MeetingParticipants.js

```diff
@@ -116,39 +120,34 @@ export function MeetingParticipantList() {
     const askUnmuteText = t('participantsPane.actions.askUnmute');
     const muteParticipantButtonText = t('dialog.muteParticipantButton');
 
-    const renderParticipant = id => (
-        <MeetingParticipantItem
-            askUnmuteText = { askUnmuteText }
-            isHighlighted = { raiseContext.participantID === id }
-            key = { id }
-            muteAudio = { muteAudio }
-            muteParticipantButtonText = { muteParticipantButtonText }
-            onContextMenu = { toggleMenu(id) }
-            onLeave = { lowerMenu }
-            participantActionEllipsisLabel = { participantActionEllipsisLabel }
-            participantID = { id }
-            youText = { youText } />
-    );
-
-    const items = [];
-
-    localParticipant && items.push(renderParticipant(localParticipant?.id));
-    participants.forEach(p => {
-        items.push(renderParticipant(p?.id));
-    });
-
     return (
     <>
         <Heading>{t('participantsPane.headings.participantsList', { count: participantsCount })}</Heading>
         {showInviteButton && <InviteButton />}
         <div>
-            { items }
+            <MeetingParticipantItems
```

Comment:
i know this is adding over existing functionality, but some of these props make no sense to be externally provided